base/applications/regedit/lang/ro-RO.rc

```diff
@@ -32,16 +32,16 @@ Opțiuni:\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /C             Importă conținutul unui fișier de regiștri.\n\
```

Comment:
Since it's clear what the intended changes are, I dare to suggest you to extend the scope of your modifications to include this line too.